azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -371,9 +375,11 @@ public void setJobOverrideProperty(final Project project, final Flow flow, final
     final String diffMessage = PropsUtils.getPropertyDiff(oldProps, prop);
     eventData.put("diffMessage", diffMessage);
     setProjectEventStatus(errorMessage, eventData);
+
+    // Send job property overridden alert
+    ExecutionControllerUtils.alertUserOnJobPropertyOverridden(project, flow, eventData, this.alerterHolder);
```

Comment:
Clarification: I believe we are not printing anything on the log because we persist in event data. 
Btw, Nice leverage of the diffMessage!

Reply:
Yes, we're not printing the metadata on the log.